website/content/docs/connect/config-entries/api-gateway.mdx

```diff
@@ -372,7 +372,7 @@ The list of references to certificates that the listener uses for TLS terminatio
 
 ### `Listeners[].TLS.Certificates[].Name`
 
-Specifies the name of the file system or inline certificate that the listener uses for TLS termination.
+Specifies the name of the [file system](/consul/docs/connect/config-entries/file-system-certificate) or [inline certificates](/consul/docs/connect/config-entries/inline-certificate) that the listener uses for TLS termination.
```

Comment:
This looks a little weird in the preview, should it be just [inline] as opposed to [inline certificates]?
<img width="683" alt="image" src="https://github.com/hashicorp/consul/assets/13151671/337fc353-140d-4236-a6ce-748a17a5b908">